playwright-e2e/tests/dsm/atcp/atcp-receive-kit.spec.ts

```diff
@@ -11,129 +11,47 @@ import ParticipantPage from 'dsm/pages/participant-page/participant-page';
 import AtcpSearchPage, { SearchByField } from 'dsm/pages/samples/search-page';
 import { WelcomePage } from 'dsm/pages/welcome-page';
 import Radiobutton from 'dss/component/radiobutton';
-import { test } from 'fixtures/dsm-fixture';
 import { getUtcDate } from 'utils/date-utils';
 import { generateAlphaNumeric } from 'utils/faker-utils';
-import { logError, logGenomeStudySampleKitReceived, logInfo } from 'utils/log-utils';
 import { studyShortName, waitForNoSpinner, waitForResponse } from 'utils/test-utils';
+import { logInfo } from 'utils/log-utils';
 
 test.describe('Receive Genome Study Kit', () => {
   const studies = [StudyEnum.AT];
   for (const study of studies) {
     let newBarcode = generateAlphaNumeric().toUpperCase();
 
     let shortId: string;
-    let guid: string;
 
     let participantPage: ParticipantPage;
     let participantListPage: ParticipantListPage;
     let navigation: Navigation;
 
-    // Fallback to use non-e2e participant if all e2e participants have kit received before
-    const nonE2EParticipants: string[] = [];
-
     test.beforeEach(async ({page, request}) => {
       navigation = new Navigation(page, request);
       const welcomePage = new WelcomePage(page);
       await welcomePage.selectStudy(study);
     });
 
     test(`Receive genome sample kit for ${study} @dsm @${study} @functional`, async ({page}) => {
-      // Instead using UI table filter and search, it is much quicker and more accurate to intercept DSM API request to find the right participant to use.
-      // Find a Playwright test user that does not have GENOME_STUDY_SPIT_KIT_BARCODE.
-      await page.route('**/*', async (route, request): Promise<void> => {
```

Comment:
This is not working because response only contains participants on page 1, not all paricipants.